neofetch

```diff
@@ -1163,10 +1153,9 @@ get_cpu() {
         ;;
 
         "FreeMiNT")
-            cpu="$(cat /kern/cpuinfo | awk -F':' '/CPU:/ {printf $2}')"
-            speed="$(cat /kern/cpuinfo | awk -F':' '/Clocking:/ {printf $2}' | awk -F'.' '{ printf $1 }')"
+            cpu="$(awk -F':' '/CPU:/ {printf $2}' /kern/cpuinfo)"
```

Comment:
These two `awk | /kern/cpuinfo` can probably become one statement. See the example in one of my other comments.